src/docs/product/performance/performance-at-scale/index.mdx

```diff
@@ -50,22 +48,15 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
-### Starred Transactions
-
-Typically, for most projects there are only a few transactions that you really care about. We prioritize [starred transactions](/product/performance/transaction-summary/#starring-key-transactions) over others, so you can have more visibility into your important data.
-
-![Starred Transactions](starred_transactions.png)
-
 ### Health Checks
 
 _Health check_ type transactions, while important for checking the stability of your application, don't have any value for you beyond the task associated with them. For this reason, we keep fewer samples of these types of transactions and instead prioritize retaining more samples of your other more valuable transactions.
 
-
 ### Low Volume Projects
 
 <Note>
 
-(*) Unlike the other retention priorities, it is not possible to disable this one on a project level, because it applies across the entire organisation.
+(\*) Unlike the other retention priorities, it is not possible to disable this one on a project level, because it applies across the entire organisation.
```

Comment:
```suggestion
(*) Unlike the other retention priorities, it is not possible to disable this one on a project level, because it applies across the entire organisation.
```